tests/casper/protocols/blockchain/test_blockchain_message.py

```diff
@@ -9,6 +9,37 @@
 
 from state_languages.blockchain_test_lang import BlockchainTestLang
 
+@pytest.mark.parametrize(
+    'estimate, is_valid',
+    [
+        (None, True),
+        ('block', False),
+        (0, False),
+        (True, False),
+    ]
+)
+def test_accepts_valid_estimates(estimate, is_valid, block):
+    if estimate == 'block':
+        Block.is_valid_estimate(block) == is_valid
```

Comment:
probably `return` after this check so you aren't inadvertently testing the estimate -- `"block"` with `is_valid` -- `False`. Even though it happens to agree, it's probably not what you meant to test.

If you want to test that a string fails, I'd do a separate test case against `"some random string taht isn't block"`